lang/python/python3-package.mk

```diff
@@ -201,7 +201,7 @@ PYTHON3_PKG_HOST_PIP_INSTALL_ARGS = \
 	)
 
 define Py3Build/FindStdlibDepends
-	$(SHELL) $(python3_mk_path)python3-find-stdlib-depends.sh -n "$(PKG_NAME)" "$(PKG_BUILD_DIR)"
+	$(SHELL) $(python3_mk_path)python3-find-stdlib-depends.sh -n "$(PKG_NAME)" "$(PKG_BUILD_DIR)";
```

Comment:
hmm; are you sure that `CheckHostPipVersionMatch` isn't the one thatt needs to be fixed?

Reply:
actually, never mind;
this is good as-is here;

it seems that when i added `Py3Build/CheckHostPipVersionMatch` the concatenation of  shell commands requires this semi-colon here;

but it only occurs when `PY3=stdlib` is set